core/exception/exception_spec.rb

```diff
@@ -82,6 +40,18 @@
     raised_second.should == caught_second
   end
 
+  it "captures an exception into $!" do
+    exception = begin
+      raise
+    rescue RuntimeError
+      $!
+    end
+
+    exception.class.should == RuntimeError
+    exception.message.should == ""
+    exception.backtrace.first.should =~ /exception_spec/
+  end
+
```

Comment:
Add spec for $!.  (Should this and $@ spec be in a new file globals_spec.rb?)

Reply:
This would be best in `language/predefined_spec.rb` which already has $! specs, but it seems fine here too.